src/transactor.js

```diff
@@ -150,11 +151,10 @@ export function transactor(firebase, handlers, todoTrxRef, closedTrxRef) {
             // here, so warn user about it)
             result = JSON.parse(JSON.stringify({result}))
           } catch (e) {
-            console.log(`Error for transaction id=${id}, result cannot be saved\n
-                        to firebase (result=${result})`)
+            console.log(`Error for transaction id=${id}, result cannot be saved\n to firebase (result=${result})`)
             result = {}
           }
-          set(closedTrxRef.child(trData.frbId), {data: trData, result})
+          set(closedTrxRef.child(trData.frbId), {data: trData, ...result})
```

Comment:
No, I will not accidentally overwrite anything. As you can see above, result is either in the form {result: <handler-return-value>} or it is an empty map.

If I did not splat it here, I would get {result: {result: <handler-return-value}} or {result: {}}